_docs/templates/email-template.md

```diff
@@ -176,19 +246,16 @@ sent. This includes notification types and ticket or notification statuses.
   Regards, Best, etc. If the email is a ticket, it is ok to include Racker name
   and title.
 
-<!--<figure class="image">
-          <img src="{{site.cdn_url}}/img/layout/email-templates/Mobile%20LInks.svg">
-      </figure>
--->
-
 ### Call outs
 
 When there is some action the user needs to take it should be prominently
 displayed and stand out from the rest of the email content. Callouts will allow
 the user to scan for pertinent information quickly.
 
 <div class="hxRow"  markdown="1">
-{% column left:"hxCol-md-4" %}
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+
 -   **Required Action** - required actions consist of tickets that are waiting
```

Comment:
Capitalize `r` in `required` after the hyphen.

Reply:
updated.